lib/utils/job-utils/command-parser.js

```diff
@@ -1023,6 +1024,58 @@ function commandParserFactory(Logger, Promise, _) {
         }
     };
 
+    CommandParser.prototype[ip] = function(data) {
+        if (data.error) {
+            return Promise.resolve({ source: 'ip', error: data.error });
+        } else if (!data.stdout) {
+            return Promise.resolve({ source: 'ip', error: new Error("No data") });
+        }
+
+        try {
+            var interfaces = _.transform(data.stdout.split('\n'), function(result, line) {
+                if (line.startsWith(':', 1)) {
+                    result.push([line]);
+                } else {
+                    result[result.length - 1].push(line);
+                }
+            });
```

Comment:
you don't need to add an explicit accumulator unless you want the accumulator to be a different type than the input object, but I will add one for clarity